### PR TITLE
Change prettification logic

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: Ashish Tiwari <ashishjaitiwari15112000@gmail.com>

**Notes for Reviewers**

This PR makes sure that the terminal fields of a schema or Design are not tampered/modified.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — DISMISSED** [review]
LGTM

**Comment by leecalcote** [comment]
Where is the dictionary being applied?

File: server/handlers/component_handler.go [hunk 1]
```diff
@@ -98,7 +109,17 @@ func (h *Handler) GetMeshmodelComponentsByName(rw http.ResponseWriter, r *http.R
 		ModelName: typ,
 		Version:   v,
 	})
-	if err := enc.Encode(res); err != nil {
+	var comps []v1alpha1.ComponentDefinition
+	for _, r := range res {
+		m := make(map[string]interface{})
+		comp, _ := r.(v1alpha1.ComponentDefinition)
+		_ = json.Unmarshal([]byte(comp.Schema), &m)
+		m = k8s.Format.Prettify(m, false)
```
Comment (leecalcote) [inline server/handlers/component_handler.go:117]: I see all the Format.Prettify() set to `false`, but don't see a `k8s.Format.Prettify(m, true)`. Should there be one?

Comment (Revolyssup) [reply server/handlers/component_handler.go:117]: No. False here means the disablement of terminal value prettification

Comment (leecalcote) [reply server/handlers/component_handler.go:117]: Oh, I see. OK.

File: server/models/pattern/core/register.go [hunk 2]
```diff
@@ -398,6 +398,11 @@ func GetScopesByK8sAPIVersionKind(apiVersion, kind string) (s []ScopeCapability)
 func GetWorkloadByID(name, id string) *WorkloadCapability {
 	res := GetWorkload(name)
 	for _, f := range res {
+		m := make(map[string]interface{})
+		_ = json.Unmarshal([]byte(f.OAMRefSchema), &m)
+		m = k8s.Format.Prettify(m, false)
```
Comment (leecalcote) [inline server/models/pattern/core/register.go:403]:
```suggestion
		// prettify(m, false) False here means the disablement of prettification for terminal values in component schemas
		m = k8s.Format.Prettify(m, false)
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**